q2_metadata/_random.py

```diff
@@ -31,8 +40,10 @@ def shuffle_groups(metadata: qiime2.CategoricalMetadataColumn,
 
     for i in range(1, n_columns):
         column_id = '%s%d' % (column_name_prefix, i)
+
```

Comment:
```suggestion
```
I would remove these extra new lines - we typically like to keep the rest of the code as-is when making any new changes, just to maintain clarity on what we're changing in a given PR and why 🙂